docs/intro.md

```diff
@@ -108,33 +160,36 @@ often want to supplement the LLM with more data:
 - Non-public information, for example for summarizing a piece of text you're creating
   within you organization.
 
-These additional pieces of information are called the _contextScope_ in xef.ai, and are attached
+These additional pieces of information are called the _context_ in xef.ai, and are attached
 to every question to the LLM. Although you can add arbitrary strings to the context at any
 point, the most common mode of usage is using an _agent_ to consult an external service,
 and make its response part of the context. One such agent is `search`, which uses a web
 search service to enrich that context.
 
 ```kotlin
-import com.xebia.functional.xef.auto.*
+import com.xebia.functional.xef.reasoning.serpapi.Search
```

Comment:
for this, we need to set up an environment variable called: `SERP_API_KEY`. Perhaps we should add some information about it

Reply:
Add it! Thanks @Montagon 👏